src/App/Containers/Authentication/Login/LoginContainer.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import LoginModule from '../Login/LoginPage'
+import { isLogin } from '../../../../Helper/LocalStorage'
+
+
+class LoginContainer extends React.PureComponent {
+
+    state = {
+        isLogin: false,
+        redirect: false,
+        redirectFromLogin: false
+    }
+
+    componentDidMount() {
+        const { history } = this.props
+        if (isLogin()) {
+            history.push('/posts')
+        }
+    }
+
+    onChangeloginHandler = (event) => {
+        let value = event.target.value
+        let fieldName = event.target.name
+
+        this.setState({
+            [fieldName]: value //this value will assign to this respective fieldname
+        })
+    }
+
+    onLoginSubmitBtn = async (e) => {
+        e.preventDefault();
+
+        const TOKEN_KEY = 'jwt'
+        const { history } = this.props
+        try {
+            const response = await fetch("http://localhost:5000/api/login", {
```

Comment:
Prefer to use .env for api urls and any other application wide constants to avoid hardcodiings.